editor/sidebar/index.js

```diff
@@ -32,6 +32,7 @@ export default connect(
 	( state ) => {
 		return {
 			panel: getActivePanel( state ),
+			selectedBlockCount: getSelectedBlockCount( state ),
```

Comment:
Should we connect the `Header` and the `BlockInspector` instead since this is not directly used here?

Reply:
Sure :)